src/RuleSetInterface.php

```diff
@@ -30,7 +30,7 @@ public static function create(array $set = array());
      *
      * @param string $rule name of rule
      *
-     * @return array|null
+     * @return array|bool
```

Comment:
@localheinz 
Don't get me wrong. I see your point about DX and I agree with it.
I just don't find changing `RuleSet` a valid solution.